pkg/connector/slack/v0/config/definition.json

```diff
@@ -15,7 +15,7 @@
       "additionalProperties": false,
       "properties": {
         "token": {
-          "description": "Fill your token",
+          "description": "Please go to the Slack API tutorial to create a Slack App, set the scope of token, and get the token.",
```

Comment:
Could you the title `s/token/Token` on L27? I saw it on the console today and it isn't aligned with other component fields.

Reply:
What does L27 mean here?

Reply:
Sorry, it means line 27. I can't add a comment there because it isn't part of the changes, but I mentioned [it in the ticket](https://linear.app/instill-ai/issue/INS-4686/improve-slack-document#comment-4b0f64f7) when checking how the console rendered these fields.